### ISSUE TITLE
VOCGTDataSet

### ISSUE THREAD

**Issue by LeiyuanMa** [hfslyc/AdvSemiSeg#23]
I don't understand why you use the VOCGTDataSet instead of VOCDataSet , they choose the same data from voc train dataset :train_ids[:partial_size],,,I think the difference between these two class is whether to pad the image and label,
first:
      ignore_mask = (labels.numpy() == 255)
      ignore_mask_gt = (labels_gt.numpy() == 255)
then:
# train with pred
D_out = interp(model_D(F.softmax(pred)))
loss_D = bce_loss(D_out, make_D_label(pred_label, ignore_mask))
# train with gt
D_gt_v = Variable(one_hot(labels_gt)).cuda(args.gpu)
D_out = interp(model_D(D_gt_v))
loss_D = bce_loss(D_out, make_D_label(gt_label, ignore_mask_gt))

I think these two loss_D is already have different D_out,is it necessary to distinguish  VOCGTDataSet and VOCDataSet ?

**Comment by hfslyc** [comment]
thanks for pointing out. Yes, they can be reduced to one dataset in this repo.

**Comment by LeiyuanMa** [comment]
Thank  you ~~~
